iFixit/react-commerce
# Vulcan: Mark guide image data as optional

Turns out that there might not be an image for a guide. This handles that case.

## QA Notes
1. Create a guide with no image (e.g., a new guide; make sure it's published).
2. Link the guide from a troubleshooting solution
3. Check that the page doesn't 500.

Here's an example that I've already put together:
https://react-commerce-git-troubleshooting-fix-500-on-noi-902ded-ifixit.vercel.app/Vulcan/imageless-guide-link

Closes https://github.com/iFixit/ifixit/issues/48470
